ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -40,6 +40,14 @@ ActivityNotificationBase {
             colorId: Utils.colorIdForPubkey(contactId)
             colorHash: Utils.getColorHashAsJson(contactId, sender.isEnsVerified)
         }
+        contentType: notification && notification.message ? notification.message.contentType : StatusMessage.ContentType.Unknown
+        album: notification && notification.message ? notification.message.albumMessageImages.split(" ") : []
```

Comment:
Are you sure it can't contain a space character?

Reply:
it must cos I did it :) its a one way how to send `seq[string]` from nim layer